src/app/shared-component/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,18 @@
+						
+<div *ngIf="selected" class="status">
```

Comment:
The span part is the label of the selected option. Clicking on which it'll show up the list of options, which is the ul part.